net/strongswan/files/swanctl.init

```diff
@@ -564,7 +564,7 @@ config_remote() {
 	swanctl_xappend3 "auth = $auth_method"
 	[ -n "$remote_identifier" ] && swanctl_xappend3 "id = \"$remote_identifier\""
 	[ -n "$remote_ca_certs" ] && swanctl_xappend3 "cacerts = \"$remote_ca_certs\""
-	[ "$auth_method" = eap-mschapv2 ] && swanctl_xappend3 "eap_id = $eap_id"
+	[ "$auth_method" = "eap-mschapv2" ] || [ "$auth_method" = "eap-tls" ] && swanctl_xappend3 "eap_id = $eap_id"
```

Comment:
Although AFAICT this is correct, there are a few alternatives to make this less ambiguous:

```suggestion
	if [ "$auth_method" = "eap-mschapv2" ] || [ "$auth_method" = "eap-tls" ]; then
		swanctl_xappend3 "eap_id = $eap_id"
	fi
```

or:

```suggestion
	case "$auth_method" in
		eap-mschapv2 | eap-tls)
			swanctl_xappend3 "eap_id = $eap_id"
			;;
	esac
```

or:

```suggestion
	[[ "$auth_method" = "eap-mschapv2" || "$auth_method" = "eap-tls" ]] && swanctl_xappend3 "eap_id = $eap_id"
```

This applies to the other case as well.

Reply:
Actually, I don't think `ash`/`Busybox` supports `[[` but I could be wrong... Try it.  Otherwise use `{ ... }` for grouping like:

```
{ [ "$auth_method" = "eap-mschapv2" ] || [ "$auth_method" = "eap-tls" ] } && swanctl_xappend3 "eap_id = $eap_id"
```